lib/licensed/source/pip.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+require "json"
+require "English"
+
+module Licensed
+  module Source
+    class Pip
+      def initialize(config)
+        @config = config
+      end
+
+      def type
+        "python"
+      end
+
+      def enabled?
+        @config.enabled?(type) && File.exist?(@config.pwd.join("requirements.txt"))
+      end
+
+      def dependencies
+        packages = parse_requirements_txt
+
+        @dependencies = packages.map do |package_name|
+            package = package_info(package_name)
+            location = File.join(package["Location"], "-" + package["Version"] + ".dist-info")
+            Dependency.new(location, {
+              "type"        => type,
+              "name"        => package["Name"],
+              "summary"     => package["Summary"],
+              "homepage"    => package["Home-page"],
+              "version"     => package["Version"]
+            })
+        end
+      end
+
+      # Build the list of packages from a 'requirements.txt'
+      # Assumes that the requirements.txt follow the format pkg=1.0.0 or pkg==1.0.0
+      def parse_requirements_txt
+        packages = []
+        File.open(@config.pwd.join("requirements.txt")).each do |line|
```

Comment:
nit: could use `map` here instead of `each` to get rid of the declaration and usage of `packages`